src/ucp/proto/proto_select.c

```diff
@@ -300,18 +277,9 @@ ucp_proto_select_perf_ranges_cleanup(ucp_proto_perf_range_t *perf_ranges,
     }
 }
 
-void ucp_proto_select_caps_reset(ucp_proto_caps_t *caps)
-{
-    caps->cfg_thresh   = UCS_MEMUNITS_AUTO;
-    caps->cfg_priority = 0;
-    caps->min_length   = 0;
-    caps->num_ranges   = 0;
-}
-
 void ucp_proto_select_caps_cleanup(ucp_proto_caps_t *caps)
 {
     ucp_proto_select_perf_ranges_cleanup(caps->ranges, caps->num_ranges);
-    ucp_proto_select_caps_reset(caps);
```

Comment:
why not cleanup/reset `min_length` and `num_ranges`?

Reply:
wanted to remove ucp_proto_select_caps_reset function altogether; it's not really mandatory to reset the values during cleanup